ArcGISRuntimeSDKQt_SampleViewers/ArcGISRuntimeSDKQt_Samples/qml/CategoryGridView.qml

```diff
@@ -49,6 +50,10 @@ Page {
             font.pixelSize: 14
             placeholderText: qsTr("Find a sample...")
             padding: 10
+
+            onTextChanged: {
+                searchQueryEventTimer.start();
```

Comment:
❓ I wonder if there is a better way to globally enable/disable this code from running? Currently the check is in post event in C++ and the network calls won't go out if it is disabled. However, all of this overhead is still needlessly running when disabled